<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoDialogClass</class>
 <widget class="QDialog" name="AutoDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="Money.qrc">
    <normaloff>:/Images/add_user.png</normaloff>:/Images/add_user.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDateEdit" name="dateEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="6">
    <layout class="QVBoxLayout">
     <item>
      <widget class="QPushButton" name="buttonOK">
       <property name="icon">
        <iconset resource="Money.qrc">
         <normaloff>:/Images/apply.png</normaloff>:/Images/apply.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="icon">
        <iconset resource="Money.qrc">
         <normaloff>:/Images/editdelete.png</normaloff>:/Images/editdelete.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>周期</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBoxInterval">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>每天</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>每周</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>每月</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>每年</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>自定义</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxMonths">
       <property name="suffix">
        <string> 个月</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>金额</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>类别</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="CategoryComboBox" name="comboBoxCategory">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>用户</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="CategoryComboBox" name="comboBoxUser">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>免提示</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CategoryComboBox</class>
   <extends>QComboBox</extends>
   <header>CategoryComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Money.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonOK</sender>
   <signal>clicked()</signal>
   <receiver>AutoDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>-23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>AutoDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
